css/styles.less

```diff
@@ -1,108 +1,186 @@
 @font-headings: 'Raleway';
 @font-paragraphs: 'Montserrat';
-@btn-width: 12rem;
-@orange: orange;
-@red: red;
-@blue: rgb(98, 98, 163);
+@pg-max-width: 1390px;
+@btn-width: 350px;
+@nav-height: 100px;
+@section-vert-gap: 80px;
+@subcontent-vert-gap: 30px;
+@orange: rgb(255, 207, 81);
+@red: rgb(232, 52, 58);
+@lightblue: rgb(85, 97, 123);
+@darkblue: rgb(33, 58, 95);
 
 * {
   box-sizing: border-box;
+  font-family: @font-paragraphs;
+  margin: 0 auto;
+  padding: 0;
 }
 
 body, html {
-  margin: 0;
-  padding: 0;
-  font-size: 16px;
+  // margin: 0;
+  // padding: 0;
+  font-size: 100%;
+  // line-height: 150%;
 }
 
-h1, h2, h3, h4, h5, h6, button, a {
+.wrapper {
+  max-width: @pg-max-width;
+  margin: 0 auto;
+}
+
+h2, h3, h4, h5, h6, button, a {
   font-family: @font-headings;
   text-transform: uppercase;
   text-align: center;
+  margin-bottom: @subcontent-vert-gap;
 }
 
-button {
-  width: @btn-width;
-  border: none;
-  background-color: unset;
-  padding: .5rem;
+h2 {
+  font-size: 64px;
+  margin-bottom: @subcontent-vert-gap + 10px;
+}
+
+h3 {
+  font-size: 48px;
+}
+
+h4 {
+  font-size: 32px;
+}
+
+p, input, select, textarea {
+  font-size: 21px;

```

Comment:
you should use em not px for responsive. alternatively vw... but that gets tricky...